apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnection.cs

```diff
@@ -121,16 +121,16 @@ public override string ConnectionString
         public ChannelCredentials GetCredentials() => _connectionStringBuilder?.GetCredentials();
 
```

Comment:
Probably remove the null check here as well?

Reply:
Done - I hadn't done it before because in my "working code" this property is gone entirely...

Reply:
OK